<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mianowanie</class>
 <widget class="QDialog" name="Mianowanie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="PanelZapytania">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:14pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskSkoczek">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: #006699;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Zasoby.qrc">
     <normaloff>:/zdjecia/skoczek_bialy.png</normaloff>:/zdjecia/skoczek_bialy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskGoniec">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>149</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: #006699;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Zasoby.qrc">
     <normaloff>:/zdjecia/goniec_bialy.png</normaloff>:/zdjecia/goniec_bialy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskWieza">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: #006699;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Zasoby.qrc">
     <normaloff>:/zdjecia/wieza_biala.png</normaloff>:/zdjecia/wieza_biala.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskHetman">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>149</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: #006699;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Zasoby.qrc">
     <normaloff>:/zdjecia/hetman_bialy.png</normaloff>:/zdjecia/hetman_bialy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
